src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,481 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const singleNode = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'single',
+  response: {
+    transition: '_success',
+    invocations: {},
+  },
+};
+
+export const compositeWithoutSubtasksField = {
+  id: 'A',
+  label: 'A label',
+  status: 'unprocessed',
+  type: 'composite',
+  response: {
+    invocations: {},
+  },
+};
+
+export const compositeNodeWithNoSubtasks = {
+  id: 'A',
+  label: 'A label',
+  status: 'unprocessed',
+  type: 'composite',
+  subtasks: [],
+  response: {
+    invocations: {},
+  },
+};
+
+export const compositeNode = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  ],
+  response: {
+    invocations: {},
+  },
+};
+
+export const compositeNodeWithTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  ],
+  response: {
+    transition: '_success',
+  },
+  on: {
+    _success: {
+      id: 'A A',
+      label: 'A A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  },
+};
+
+export const compositeWithComplexTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'error',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      on: {
+        _error: {
+          id: 'A-B A',
+          label: 'A-B A label',
+          status: 'success',
+          type: 'single',
+          response: {
+            invocations: {},
+          },
+        },
+      },
+      response: {
+        invocations: {},
+        transition: '_error',
+      },
+    },
+  ],
+  on: {
+    _success: {
+      id: 'B',
+      label: 'B label',
+      status: 'error',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  },
+  response: {
+    invocations: {},
+    transition: '_success',
+  },
+};
+
+export const singleNodeWithTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
```

Comment:
```suggestion
  status: 'other',
```
 other - a node ends with some custom transition